### DISCUSSION TITLE
Turn off thumbnails

### DISCUSSION THREAD

**Discussion by sky-cake** [immich-app/immich#12400] [General]
Hello, I do not want thumbnails to be used or generated. How can I turn them off?

**Comment by alextran1502** [comment]
There is no explicit option, but you can pause the thumbnail generation job and then clear the queue after everything is uploaded.

Note that most if not all of the features rely on generated thumbnail

**Reply by sky-cake** [reply]
Okay, thanks. I realize I am looking for a heavily watered down web image viewer.

I just built my own with Flask since I every other solution (photoviewer, lychee, homegallery, etc.) all seem to generate thumbnails, and I don't want that overhead.
